identity-service/src/routes/healthCheck.js

```diff
@@ -10,7 +10,7 @@ const axios = require('axios')
 let notifDiscProv = config.get('notificationDiscoveryProvider')
 
 // Defaults used in relay health check endpoint
-const RELAY_HEALTH_ONE_HOUR_AGO_BLOCKS = 720 // 5 blocks/sec = 720 blocks/hr
+const RELAY_HEALTH_ONE_HOUR_AGO_BLOCKS = 120 // 1 block/5sec = 120 blocks/hr
```

Comment:
@vicky-g can you also change the max block range to like 200 below? and there's a few comments that should be updated too